src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_BOUND = 100;
+    private Random randomNumber = new Random();
```

Comment:
Avoid using static methods in your solution. Instead of using a static `Random` instance, you should create an instance of `Lottery` and use its methods. This will make your code more object-oriented.